controllers/network/bgpconfiguration_controller.go

```diff
@@ -243,38 +243,61 @@ func (r *BGPConfigurationReconciler) SetupWithManager(ctx context.Context, mgr c
 			// Skip if
 			// * no NAD annotation was configured on old object OR
 			// * no NAD annotation was configured on new object AND
+			// * no predictableip label was configured on old object OR
+			// * no predictableip label was configured on new object AND
 			// * the resourceVersion has not changed
 
-			oldConfigured := true
+			oldNADConfigured := true
 			if val, ok := e.ObjectOld.GetAnnotations()[k8s_networkv1.NetworkAttachmentAnnot]; !ok || len(val) == 0 {
-				oldConfigured = false
+				oldNADConfigured = false
 			}
-			newConfigured := true
+			newNADConfigured := true
 			if val, ok := e.ObjectNew.GetAnnotations()[k8s_networkv1.NetworkAttachmentAnnot]; !ok || len(val) == 0 {
-				newConfigured = false
+				newNADConfigured = false
 			}
 
-			return (oldConfigured || newConfigured) && e.ObjectOld.GetResourceVersion() != e.ObjectNew.GetResourceVersion()
+			oldPredictableIPConfigured := true
+			if val, ok := e.ObjectOld.GetLabels()["predictableip"]; !ok || len(val) == 0 {
```

Comment:
`predictableip` is used outside this operator and set on the pods, right? might be good to make this a const in the api module that it can be used from other service operators.